resources/aws/kms.go

```diff
@@ -55,6 +55,12 @@ func (kk *KMSKey) CreateOrFail() error {
 		return microerror.Mask(err)
 	}
 
+	if _, err := kk.Clients.KMS.EnableKeyRotation(&kms.EnableKeyRotationInput{
+		KeyId: aws.String(*key.KeyMetadata.KeyId),
```

Comment:
You can use `key.KeyMetadata.KeyId` here without the `aws.String` helper. As `KeyId` is already a string pointer.

Reply:
ok thx for the comment